### ISSUE TITLE
Color contrast on Edit Work Type Page

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3083]
vpat.hykudemo.org
Version 7.1

### Descriptive summary
Color contrast for the language and login drop-down menus is not clear enough:
<img width="2023" height="1043" alt="Image" src="https://github.com/user-attachments/assets/d50223ba-b2fc-46bc-b38e-55f6080c6053" />

### Rationale
Accessibility issue remediation

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

### Expected behavior
Color contrast should be at least 4.5: 1

### Actual behavior
Color contrast is 4.2:1

### Steps to reproduce the behavior
- Open the page
- Use the SiteImprove browser extension

### Related
https://github.com/samvera/hyku/issues/3075
